bindings/pyroot/pythonizations/test/rdf_filter_pyz_helper.py

```diff
@@ -0,0 +1,128 @@
+import ROOT
+import numpy as np
+
+def CreateData():
+    """
+    This function generates the root files of various datatypes with random values to test them.
+    Not all datatypes could be generated such as.
+        1. Strings
+        2. Char_t
+        3. UChar_t
+
+    Cannot test String, Char_t and UChar_t right now as:
+    1. The generation of it (which I assumed would be equivalent to 8bit integers is false).
+    2. They cannot be converted into numpy arrays thus hindering my current "test conditions" which takes the mean of the entries.
```

Comment:
This information is not fit for a class docstring. I'm not sure whether we can put it as a comment just before the call to `Snapshot`, or more probably we should just take note of this somewhere else, not in actual code